src/cascadia/TerminalCore/Terminal.hpp

```diff
@@ -153,6 +153,7 @@ class Microsoft::Terminal::Core::Terminal final :
     bool _selectionActive;
     SHORT _selectionAnchor_YOffset;
     SHORT _endSelectionPosition_YOffset;
+    std::wstring_view _doubleClickDelimiters;

```

Comment:
@carlos-zamora using string_view for storage is often questionable.  Are you sure the allocated string outlives the string_view?   In my opinion it's safer to use the std::wstring.

Reply:
This is true. If you're going to hold the delimiters in a persistent manner, make a copy with a `std::wstring` instead.